docs/networking/networking.md

```diff
@@ -76,6 +83,14 @@ To disable the embedded LB, configure all servers in the cluster with the `--dis
 
 This is necessary if you wish to run a different LB, such as MetalLB.
 
+## External Cloud Controller Manager
+
+K3s ships with a stub Cloud Controller Manager (CCM) that does not actually implement any functionality. In order to use an external CCM, you must start K3s with the `--disable-cloud-controller` flag. 
```

Comment:
This isn't accurate. While it does not actually interface with any infrastructure providers, our stub CCM handles several critical operations:
* Setting node InternalIP and ExternalIP address fields based on the --node-ip and --node-external-ip flags.
* Hosting the ServiceLB LoadBalancer controller
* Clearing the `node.cloudprovider.kubernetes.io/uninitialized` taint that is present when the cloud-provider is set to `external` (which is the only valid value, since we disable all the built-in cloud providers).

If users disable the stub CCM, they must deploy an alternative CCM, or their nodes will remain tainted and unschedulable.

Reply:
Thanks for that info, I have incorporated it into the section.